javascript/newer-JS-features.md

```diff
@@ -247,6 +425,45 @@ const produce = [];
 console.log(produce);
 ```
 
+What will the console.log output after running the following code?
+
+```javascript
+const message = "The rain in Spain falls mainly in the plain.";
+const updatedMessage = message.replaceAll("in", "on");
+console.log(updatedMessage);
+```
+1.	“The rain on Spaon falls maonly on the plaon.”
+2.	**“The raoon oon Spaon falls maonly oon the plaon.”**
+3.	“The rain in Spain falls mainly in the plain.”
+4.	“The raon on Spon falls monly on the plon.”
+
+What will the console.log output after running the following code?
+
+```javascript
+const double = x => x * 2;
+const increment = x => x + 1;
+
+const result = 4 |> double |> increment;
+console.log(result);
+```
+
+1.	5
+2.	8
+3.	**9**
+4.	10
+
+What will the console.log output when this code is run?
+
+```javascript
+const now = Temporal.Now.plainDateTimeISO();
+console.log(now.toString());
+```
+
+1.	**“2024-12-23T10:35:00” (or your current date and time in ISO format).**
```

Comment:
It's bolded because it's the answer to the multiple choice question. I followed the bolding format of the original file as the first question is multiple choice and the answer is bolded as well!

<img width="603" alt="Screenshot 2024-12-26 at 7 03 16 PM" src="https://github.com/user-attachments/assets/1dc9e83a-20e7-4b6e-8d2b-4e9e8cb7b943" />